apps/src/localization/Localize.ts

```diff
@@ -0,0 +1,355 @@
+/**
+ * The possible options that can be given to the LocalizeJS widget and the
+ * possible argument in the 'initialize' event callback.
+ */
+export type LocalizeOptions = {
```

Comment:
This is mostly just the type information for the `Localize` widget that will be installed in the background when we hook up LocalizeJS properly in a subsequent PR.

Reply:
I believe `@types/localizejs-library` can be added as a dev dependency to set this type globally?

Reply:
Interesting, can we request Localize to update them?